src/screenshots/capturer.js

```diff
@@ -147,7 +147,11 @@ export default class Capturer {
             await generateThumbnail(screenshotPath, thumbnailPath);
         });
 
+        const testRun   = this.testEntry.testRuns[this.browserId];
+        const testRunId = testRun && testRun.id;
```

Comment:
Can it be simplified as `const testRunId = this.testEntry.testRuns[this.browserId].id`?

Reply:
Right now, I cannot imagine the situation when testRun can be null in this code, so I guess, we can simplify it